### PR TITLE
UI check

### PR DESCRIPTION
**Pull request by matiduda**
The UI part is mostly done. I implemented it according to the specification. Please check the About page for any errors. To make it easier for you here is the website running on my network locally:

screenshots

Now we can focus event handlers and making sure the network runs correctly :)

### CODE REVIEW

**Comment by MrDell1** [comment]
The top and bottom of Card component is cut 
image
image

**Comment by MrDell1** [comment]
I think the About and Reward(button to Reedem should be disable) page should be visible even if user is no connetcted
image

**Comment by MrDell1** [comment]
There the content is outside the box 
image

File: frontend/src/routes/ContentWraper/MainBox/CardWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { AwardCard } from "./AwardCard";
+
+export const CardWrapper = (): ReactElement => {
+    return (
+        <>
+            <Container maxW="150ch" centerContent>
+                <Wrap spacing='70px' justify='center'>
+                    <WrapItem>
+                        <Center w='300px' h='300px'>
+                            <AwardCard title="Allegro 50PLN" value="5,00" description="Allegro gift card" image="https://a.allegroimg.com/s720/03444d/5fbfc8ff4da986b370a7d62997c3/Karta-Podarunkowa-Prezent-50-zl" />
```
Comment (MrDell1) [inline frontend/src/routes/ContentWraper/MainBox/CardWrapper.tsx]: Make const for title, value, description and img

File: frontend/src/routes/ContentWraper/MainBox/AwardCard.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+const currency = 'MHT';
+
+export const AwardCard = ({ title = 'Award', description = 'Awesome award', value = '0,00', image }: Props): ReactElement => {
+    return (
+        <Card maxW='sm'>
+            <CardBody>
+                <Image
+                    src={image}
+                    alt='Green double couch with wooden legs'
```
Comment (MrDell1) [inline frontend/src/routes/ContentWraper/MainBox/AwardCard.tsx]: Get altText from Parent(like image, value, etc.)

File: frontend/src/routes/ContentWraper/MainBox/AwardCard.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+type Props = {
+    title: string;
+    description: string;
+    value: string;
+    image: string;
+}
+
+const currency = 'MHT';
+
+export const AwardCard = ({ title = 'Award', description = 'Awesome award', value = '0,00', image }: Props): ReactElement => {
```
Comment (MrDell1) [inline frontend/src/routes/ContentWraper/MainBox/AwardCard.tsx]: We don't need default values for title, description.... . If we display the component, we always know these values.

File: frontend/src/routes/ContentWraper/MainBox/CardWrapper.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import { ReactElement } from "react";
+import { AwardCard } from "./AwardCard";
+
+export const CardWrapper = (): ReactElement => {
+    return (
+        <>
+            <Container maxW="150ch" centerContent>
+                <Wrap spacing='70px' justify='center'>
+                    <WrapItem>
+                        <Center w='300px' h='300px'>
```
Comment (MrDell1) [inline frontend/src/routes/ContentWraper/MainBox/CardWrapper.tsx]: I think w and h are a problem with clipped edges.

**Review by MrDell1 — CHANGES_REQUESTED** [review]

File: frontend/src/routes/ContentWraper/MainBox/AwardCard.tsx (hunk 2 above)
Comment (matiduda) [reply frontend/src/routes/ContentWraper/MainBox/AwardCard.tsx]: Nice suggestion!

File: frontend/src/routes/ContentWraper/MainBox/CardWrapper.tsx (hunk 4 above)
Comment (matiduda) [reply frontend/src/routes/ContentWraper/MainBox/CardWrapper.tsx]: I added a spacer at the bottom and it solved this issue

**Review by matiduda — COMMENTED** [review]
I've resolved requested changes, commiting now

File: frontend/src/routes/ContentWraper/MainBox/CardWrapper.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                    <WrapItem>
+                        <Center>
+                            <AwardCard
+                                title={cardData.title}
+                                value={cardData.value}
+                                description={cardData.description}
+                                image={cardData.image} />
+                        </Center>
+                    </WrapItem>
+                    <Box h='100px' w='100%' />
```
Comment (MrDell1) [inline frontend/src/routes/ContentWraper/MainBox/CardWrapper.tsx]: I think this is useless here

File: frontend/src/routes/Root/MainBox/CardWrapper.tsx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+                image={cardData.image}
+                title={cardData.title}
+                value={cardData.value}
+              />
+            </Center>
+          </WrapItem>
+          <Box h="100px" w="100%" />
+        </Wrap>
+      </Container>
+    </>
```
Comment (MrDell1) [inline frontend/src/routes/Root/MainBox/CardWrapper.tsx]: Use this 
``` return(
<Flex flexWrap="wrap" gap="8" justify="center">
      <AwardCard
        description={cardData.description}
        image={cardData.image}
        title={cardData.title}
        value={cardData.value}
      />

      <AwardCard
        description={cardData.description}
        image={cardData.image}
        title={cardData.title}
        value={cardData.value}
      />

      <AwardCard
        description={cardData.description}
        image={cardData.image}
        title={cardData.title}
        value={cardData.value}
      />

      <AwardCard
        description={cardData.description}
        image={cardData.image}
        title={cardData.title}
        value={cardData.value}
      />

      <AwardCard
        description={cardData.description}
        image={cardData.image}
        title={cardData.title}
        value={cardData.value}
      />

      <AwardCard
        description={cardData.description}
        image={cardData.image}
        title={cardData.title}
        value={cardData.value}
      />
    </Flex>
);
```

**Review by MrDell1 — CHANGES_REQUESTED** [review]

**Review by matiduda — COMMENTED** [review]
Done

**Review by MrDell1 — APPROVED** [review]

**Pull request merged**